adminpages/reports/sales.php

```diff
@@ -401,79 +412,133 @@ function pmpro_ShowMonthOrYear()
-
-			var data = google.visualization.arrayToDataTable([
-				[
-					'<?php echo esc_html( $date_function );?>', '<?php _e( 'Renewals', 'paid-memberships-pro' );?>', '<?php echo esc_html( sprintf( __( 'New %s', 'paid-memberships-pro' ), ucwords( $type ) ) );?>', '<?php _e( 'Average*', 'paid-memberships-pro' );?>', 
-				],
-				<?php foreach($cols as $date => $value) { 
-					?>
-					['<?php
-						if ( $period == "monthly" ) {
-							echo esc_html(date_i18n("M", mktime(0,0,0,$date,2)));
-						} else {
-							echo esc_html( $date );
-						}
-					?>', <?php echo esc_html( pmpro_round_price( $value[1] ) );?>, <?php echo esc_html( pmpro_round_price( $value[0] - $value[1] ) );?>, <?php echo esc_html( pmpro_round_price( $average ) );?>,  ], 
+			var dataTable = new google.visualization.DataTable();
+			dataTable.addColumn('string', '<?php echo esc_html( $date_function ); ?>');
+			dataTable.addColumn({type: 'string', role: 'tooltip', 'p': {'html': true}});
+			dataTable.addColumn('number', '<?php esc_html_e( 'Renewals', 'paid-memberships-pro' ); ?>');
+			dataTable.addColumn('number', '<?php esc_html_e( sprintf( __( 'New %s', 'paid-memberships-pro' ), ucwords( $type ) ) ); ?>');
+			<?php if ( $type === 'sales' ) { ?>
+				dataTable.addColumn('number', '<?php esc_html_e( sprintf( __( 'Average: %s', 'paid-memberships-pro' ), number_format_i18n( $average, 2 ) ) ); ?>');
+			<?php } else { ?>
+				dataTable.addColumn('number', '<?php esc_html_e( sprintf( __( 'Average: %s', 'paid-memberships-pro' ), pmpro_escape_price( html_entity_decode( pmpro_formatPrice( $average ) ) ) ) ); ?>');
+			<?php } ?>
+			dataTable.addRows([
+				<?php foreach($cols as $date => $value) { ?>
+					[
+						'<?php
+							if ( $period == "monthly" ) {
+								echo esc_html(date_i18n( 'M', mktime(0,0,0,$date,2)));
+							} else {
+								echo esc_html( $date );
+							}
+						?>',
+						createCustomHTMLContent(
+							'<?php
+								if ( $period == "monthly" ) {
+									echo esc_html(date_i18n( 'F', mktime(0,0,0,$date,2)));
+								} elseif ( $period == "daily" ) {
+									echo date_i18n( get_option( 'date_format' ), strtotime( $year . '-' . $month . '-' . $date ) );
+								} else {
+									echo esc_html( $date );
+								}
+							?>',
+							<?php if ( $type === 'sales' ) { ?>
+								'<?php esc_html_e( $value[1] ); ?>',
+								'<?php esc_html_e( $value[0] - $value[1] ); ?>',
+								'<?php esc_html_e( $value[0] ); ?>',
+							<?php } else { ?>
+								'<?php echo pmpro_escape_price( pmpro_formatPrice( $value[1] ) ); ?>',
+								'<?php echo pmpro_escape_price( pmpro_formatPrice( $value[0] - $value[1] ) ); ?>',
+								'<?php echo pmpro_escape_price( pmpro_formatPrice( $value[0] ) ); ?>',
+							<?php } ?>
+						),
+						<?php echo esc_html( pmpro_round_price( $value[1] ) ); ?>,
+						<?php echo esc_html( pmpro_round_price( $value[0] - $value[1] ) ); ?>,
+						<?php echo esc_html( number_format_i18n( $average, 2 ) ); ?>,
```

Comment:
@kimcoleman These appear to be numbers that are converted to strings, but they are echo'd like numbers. Using `wp_json_encode` will echo them as strings. Please double check the output here that's being used by `pmpro_round_price` to ensure it's outputting exactly as you intend it now.

```suggestion
						<?php echo wp_json_encode( pmpro_round_price( $value[1] ) ); ?>,
						<?php echo wp_json_encode( pmpro_round_price( $value[0] - $value[1] ) ); ?>,
						<?php echo wp_json_encode( pmpro_round_price( $average ) ); ?>,
```